scalardb-cluster-standalone-mode/docker-compose.yaml

```diff
@@ -1,28 +1,52 @@
 version: "3.5"
```

Comment:
```suggestion
```
Since it is an obsolete configuration

Reply:
Thank you. Since it should also be applied to the other docker compose files in other samples, I will make the change in a separate PR. Thanks.